tests/test_SklearnSVMConverters.py

```diff
@@ -78,13 +118,70 @@ def test_convert_svmc_linear_binary(self):
             X,
             model,
             model_onnx,
-            basename="SklearnBinSVCLinearPF",
-            allow_failure=True,
+            basename="SklearnBinSVCLinearPF-NoProbOpp",
+            allow_failure="StrictVersion(onnxruntime.__version__)"
+                          " < StrictVersion('0.5.0')"
         )
 
-    def test_convert_svmc_linear_multi(self):
```

Comment:
Why were these originally named "svmc" before? I noticed the same naming structure in onnxmltools' unit tests for libSVM.

Reply:
Probably me, I changed for something more explicit.